docs/man/docker-images.1.md

```diff
@@ -34,7 +34,9 @@ versions.
    Show all images (by default filter out the intermediate image layers). The default is *false*.
 
 **-f**, **--filter**=[]
-   Provide filter values (i.e., 'dangling=true')
+   Provide filter values. Valid filters:
```

Comment:
@moxiegirl The dangling=true still needs to be in there.  We just added a new "filter".  So `--filter dangling=true` was already there but now you can do `label=...`.